test/cbmc/proofs/Posix_OtaReceiveEvent/Posix_OtaReceiveEvent_harness.c

```diff
@@ -35,6 +35,8 @@
 #include <poll.h>
 #include <mqueue.h>
 
+int nondet_int();
```

Comment:
Why does this function now need to be declared?

Reply:
It always had to be, it just seems no one cared about the warning.